data.tf

```diff
@@ -0,0 +1,19 @@
+data "aws_region" "current" {
+  current = true
+}
+
+data "aws_ami" "dp_web" {
+  most_recent = true
+
+  filter {
+    name = "name"
+
+    values = [
+      "dq-data-pipeline-server*",
+    ]
+  }
+
+  owners = [
+    "093401982388",
```

Comment:
shouldn't the owner be `self` now that it should be replicated in to the other accounts?